### PR TITLE
[DOC] Fix formatting for What's Here in IO

### PR DESCRIPTION
**Pull request by BurdetteLamar**

### CODE REVIEW

File: io.c [hunk 1]
```diff
@@ -14508,133 +14508,133 @@ set_LAST_READ_LINE(VALUE val, ID _x, VALUE *_y)
...
+ *  - #read: Returns all remaining or the next _n_ bytes read from +self+
+ *    for a given _n_.
+ *  - #read_nonblock: the next _n_ bytes read from +self+ for a given _n_,
+ *    in non-block mode.
+ *  - #readbyte: Returns the next byte read from +self+;
+ *    same as #getbyte, but raises an exception on end-of-file.
+ *  - #readchar: Returns the next character read from +self+;
+ *    same as #getc, but raises an exception on end-of-file.
+ *  - #readline: Returns the next line read from +self+;
+ *    ame as #getline, but raises an exception of end-of-file.
```
Comment (jeremyevans) [inline io.c:14539]: `s/ame/same/`

File: io.c [hunk 2]
```diff
@@ -14508,133 +14508,133 @@ set_LAST_READ_LINE(VALUE val, ID _x, VALUE *_y)
...
+ *  - #closed?: Returns whether +self+ is closed.
+ *  - #eof? (aliased as #eof): Returns whether +self+ is at end-of-file.
+ *  - #external_encoding: Returns the external encoding object for +self+.
+ *  - #fileno (aliased as #to_i): Returns the integer file descriptor for +self+
+ *  - #internal_encoding: Returns the internal encoding object for +self+.
+ *  - #pid: Returns the process ID of a child process associated with +self+,
+ *    if +self+ was created by ::popen.
+ *  - #stat: Returns the File::Stat object containing status information for +self+.
+ *  - #sync: Returns whether +self+ is in sync-mode.
+ *  - #tty?: (aliased as #isatty): Returns whether +self+ is a terminal.
```
Comment (jeremyevans) [inline io.c:14607]: Should probably add `?`, not `?:`.

**Review by jeremyevans — CHANGES_REQUESTED** [review]
Couple of minor issues, OK to commit without further review after fixing them.

File: io.c (hunk 1 above)
Comment (BurdetteLamar) [reply io.c]: Fixed.

File: io.c (hunk 2 above)
Comment (BurdetteLamar) [reply io.c]: Fixed.

**Comment by BurdetteLamar** [comment]
Pushed numeric.c, so requesting re-review.

**Review by jeremyevans — APPROVED** [review]

**Pull request merged**